swerve100/src/main/java/org/team100/frc2023/autonomous/DriveToWaypoint3.java

```diff
@@ -1,253 +1,253 @@
-        // if (m_trajectory == null) {
-        // return;
-        // }
-        // if (goalOffsetSupplier.get() != previousOffset) {
-        // m_trajectory = makeTrajectory(goalOffsetSupplier.get(),
-        // m_trajectory.sample(m_timer.get()).velocityMetersPerSecond);
-        // previousOffset = goalOffsetSupplier.get();
-        // m_timer.restart();
-        // }
-        // if (m_trajectory == null) {
-        // return;
-        // }
-        double curTime = m_timer.get();
-        State desiredState = m_trajectory.sample(curTime);
-
-        Pose2d currentPose = m_swerve.getPose();
-        Twist2d fieldRelativeTarget = m_controller.calculate(
-                currentPose,
-                desiredState,
-                m_goal.getRotation());
-
-        SwerveState manualState = SwerveDriveSubsystem.incremental(currentPose, fieldRelativeTarget);
-        m_swerve.setDesiredState(manualState);
-
-        desiredXPublisher.set(desiredState.poseMeters.getX());
-        desiredYPublisher.set(desiredState.poseMeters.getY());
-        poseXPublisher.set(m_swerve.getPose().getX());
-        poseYPublisher.set(m_swerve.getPose().getY());
-        desiredRotPublisher.set(m_goal.getRotation().getRadians());
-        rotSetpoint.set(m_rotationController.getSetpoint().position);
-        poseRotPublisher.set(m_swerve.getPose().getRotation().getRadians());
-        poseXErrorPublisher.set(xController.getPositionError());
-        poseYErrorPublisher.set(yController.getPositionError());
-    }
-
-    @Override
-    public boolean isFinished() {
-        return isFinished; // keep trying until the button is released
-    }
-
-    @Override
-    public void end(boolean interrupted) {
-        // System.out.println("END");
-        m_timer.stop();
-        m_swerve.truncate();
-    }
-
-    //////////////////////////////////////////////////////////////////////
-
-    private final NetworkTableInstance inst = NetworkTableInstance.getDefault();
-    private final NetworkTable table = inst.getTable("drive to waypoint");
-    private final DoublePublisher desiredXPublisher = table.getDoubleTopic("Desired X").publish();
-    private final DoublePublisher desiredYPublisher = table.getDoubleTopic("Desired Y").publish();
-    private final DoublePublisher poseXPublisher = table.getDoubleTopic("Pose X").publish();
-    private final DoublePublisher poseYPublisher = table.getDoubleTopic("Pose Y").publish();
-    private final DoublePublisher poseRotPublisher = table.getDoubleTopic("Pose Rot").publish();
-    private final DoublePublisher desiredRotPublisher = table.getDoubleTopic("Desired Rot").publish();
-    private final DoublePublisher poseXErrorPublisher = table.getDoubleTopic("Error X").publish();
-    private final DoublePublisher poseYErrorPublisher = table.getDoubleTopic("Error Y").publish();
-    private final DoublePublisher rotSetpoint = table.getDoubleTopic("Rot Setpoint").publish();
-
-}
+// package org.team100.frc2023.autonomous;
```

Comment:
you should be able to leave this alone if your stuff is completely separate, no?